cartographer/io/points_processor_pipeline_builder.h

```diff
@@ -23,53 +23,45 @@
 
 #include "cartographer/common/lua_parameter_dictionary.h"
 #include "cartographer/io/points_processor.h"
+#include "cartographer/mapping/proto/trajectory.pb.h"
 
 namespace cartographer {
 namespace io {
 
-// Singleton that knows how to build a points processor pipeline out of a Lua
-// configuration. All the PointsProcessor shipping with Cartographer are already
-// registered with 'instance', but can register new classes with it that must
-// define its name and a way to build itself out of a LuaParameterDictionary.
-// See the various 'PointsProcessor's for examples.
+// Builder to create a points processor pipeline out of a Lua configuration. You
```

Comment:
Builder to create a points processor pipeline out of a Lua configuration.

You can register all built-in PointsProcessors using 'RegisterBuiltInPointsProcessors'. Non-built-in PointsProcessors must define a name and a factory method for building itself from a LuaParameterDictionary. See the various built-in PointsProcessors for examples.